renderdoc/driver/d3d12/d3d12_pixelhistory.cpp

```diff
@@ -222,7 +222,7 @@ enum D3D12PixelHistoryTests : uint32_t
 namespace
 {
 
-bool IsDepthFormat(D3D12_RESOURCE_DESC desc, CompType typeCast)
+static bool IsDepthFormat(D3D12_RESOURCE_DESC desc, CompType typeCast)
```

Comment:
These free functions are in an anonymous namespace which restricts them to the file, the `static` modifier isn't needed. But if we want to be consistent between the Vulkan/D3D12 versions, we should pick one way rather than both. :)

Reply:
I will change to follow the Vulkan approach. Style wise to me they seem identical however for the compiler it will warn/error if a static function in a file is not used, whereas it does not do that for functions in a namespace. I am guessing it doesn't think they are restricted to file scope.